### DISCUSSION TITLE
Is there a way to open code note in external editor?

### DISCUSSION THREAD

**Discussion by shirobachi** [TriliumNext/Trilium#3291] [Q&A] [answered]
Hey, 

I am writing something what someday maybe I'll call plugin, is there a way to open a whole note folder in an additional editor like VS Code? I know that this won't be easy since originally it is saved in SQLite file, so I am actually asking you about your tips how to do it?

Thank in advance!

**Comment by zadam** [comment, marked as answer]
It's possible to open a single note in the default application:

![image](https://user-images.githubusercontent.com/617641/200172213-af14ef9c-9a20-47ac-8cf2-ab9ea7116bdd.png)

Opening whole folder is not possible and quite unlikely (since the structure does not map well onto filesystem).
